### DISCUSSION TITLE
[4.0] English module titles after a new Joomla installation

### DISCUSSION THREAD

**Discussion by csbalazs** [joomla/joomla-cms#47127] [2. Cold Features]
After a new installation, the titles of the default backend modules are still in English even though I set another language as the default during the installation. If I set a default language during installation (4.0.3), why aren't module titles updated in the database? 
![image](https://user-images.githubusercontent.com/1657924/133803458-447d9dd5-6d93-409d-aa38-d3bce91b2a77.png)

**Comment by zero-24** [comment]
The module titles are not handled as language string and the setting that could handle this has been redjected in the past to be enabled by default. (#21330)

So the only way you get translated titles you have to install an full package as provided by your translation team like JGerman does for German. I dont know which other TT provides such packs with a localise.sql file

**Comment by csbalazs** [comment]
The sample data is already included in language files, so I don't understand why it would be difficult to automatically change the module titles. Unfortunately I can't find the localise.sql file in the German package.

**Comment by zero-24** [comment]
> The sample data is already included in language files, so I don't understand why it would be difficult to automatically change the module titles. 

The argument was that when that option is enabled the string added to the title field itself is ignored.

>Unfortunately I can't find the localise.sql file in the German package.

Its here: https://github.com/joomlagerman/joomla/tree/4.0-dev/installation/sql/mysql

Its only used in the full package we provided within our own build releases: https://github.com/joomlagerman/joomla/releases

**Comment by richard67** [comment]
Closing as having a pull request. Please test #36954 . Thanks in advance.

**Comment by richard67** [comment]
Sorry, re-opening because the PR solves the issue only partly.

**Comment by brianteeman** [comment]
If #36954 was complete then this would be resolved

**Comment by brianteeman** [comment]
This should be closed and #36954 completed
